src/assets.php

```diff
@@ -274,3 +287,18 @@ function load_scripts(): void {
 }
 
 load_scripts();
+
+/**
+ * Set the template values for the editor.
+ */
+function set_template_values(): void {
+	$current_screen = get_current_screen();
+	if ( ( 'post' !== $current_screen->base ) || ( 'nb_newsletter' !== $current_screen->post_type ) ) {
+		return;
+	}
+	$template_id = get_post_meta( get_the_ID(), 'nb_newsletter_template', true );
+	$bg_color    = get_post_meta( $template_id, 'nb_template_bg_color', true );
+	$font        = get_post_meta( $template_id, 'nb_template_font', true );
+	$link_color  = get_post_meta( $template_id, 'nb_template_link_color', true );
+	echo( "<style>:root {--template-font-family: {$font}; --template-bg-color: {$bg_color}; --template-link-color: {$link_color};}</style>" ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
```suggestion
	printf(
		'<style>:root {--template-font-family: %s; --template-bg-color: %s; --template-link-color: %s;}</style>',
		$font,
		$bg_color,
		$link_color,
	);
```
I would feel better about this...

Reply:
I feel better when you feel better, so DONE